packages/block-library/src/categories/index.php

```diff
@@ -34,7 +34,10 @@ function render_block_core_categories( $attributes ) {
 		$id                       = 'wp-block-categories-' . $block_id;
 		$args['id']               = $id;
 		$args['show_option_none'] = __( 'Select Category' );
-		$wrapper_markup           = '<div %1$s><label class="screen-reader-text" for="' . esc_attr( $id ) . '">' . __( 'Categories' ) . '</label>%2$s</div>';
+		$show_label               = empty( $attributes['showLabel'] ) ? ' screen-reader-text' : '';
+		$default_label            = __( 'Categories' );
+		$label_text               = ! empty( $attributes['label'] ) ? $attributes['label'] : $default_label;
```

Comment:
See [comment](https://github.com/WordPress/wordpress-develop/pull/7226#discussion_r1766068455) from @peterwilsoncc while syncing packages to WordPress core:

> Escaping label text

Reply:
I will create a PR for this and add the backport to beta label.

Reply:
Thank you 👍🏻